Doc/library/configparser.rst

```diff
@@ -915,7 +918,8 @@ ConfigParser Objects
       keys and values are implicitly converted to strings.
 
    .. versionchanged:: 3.7
-      The default *dict_type* is :class:`dict`.
+      The default *dict_type* is :class:`dict`, since it now guarantees
+      insertion order.
```

Comment:
This sounds a little funny to me.  What do you think of `preserves` or `keeps` rather than `guarantees`?

Reply:
I like preserves too :)